src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4298,7 +4298,13 @@ .FORWARDHELPCATEGORY Cmdlet
         }
         else {
             $pagerCommand = 'less'
-            $pagerArgs = '-s','-P','Page %db?B of %D:.\. Press h for help or q to quit\.'
+            # PSNativeCommandArgumentPassing arguments should be constructed differently.
+            if ((Get-ExperimentalFeature PSNativeCommandArgumentPassing).Enabled) {
```

Comment:
You can use `$EnabledExperimentalFeatures` and avoid the cmdlet call

Reply:
fixed